config/samples/core_v1beta1_openstackcontrolplane_network_isolation.yaml

```diff
@@ -73,46 +93,74 @@ spec:
       memcached:
         replicas: 1
   neutron:
+    apiOverride:
+      route: {}
     template:
+      override:
+        service:
+          internal:
+            metadata:
+              annotations:
+                metallb.universe.tf/address-pool: internalapi
+                metallb.universe.tf/allow-shared-ip: internalapi
+                metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+            spec:
+              type: LoadBalancer
       databaseInstance: openstack
       secret: osp-secret
-      externalEndpoints:
-      - endpoint: internal
-        ipAddressPool: internalapi
-        loadBalancerIPs:
-        - 172.17.0.80
       networkAttachments:
       - internalapi
   horizon:
+    apiOverride:
+      route: {}
     template:
       replicas: 1
       secret: osp-secret
   nova:
+    apiOverride:
+      route: {}
     template:
       apiServiceTemplate:
-        externalEndpoints:
-        - endpoint: internal
-          ipAddressPool: internalapi
-          loadBalancerIPs:
-          - 172.17.0.80
-      secret: osp-secret
+        override:
+          service:
+            internal:
+              metadata:
+                annotations:
+                  metallb.universe.tf/address-pool: internalapi
+                  metallb.universe.tf/allow-shared-ip: internalapi
+                  metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+              spec:
+                type: LoadBalancer
```

Comment:
Is this something we expect the human user to set or is it more like something that the service operator implementation should decide?

Reply:
the human/user, this moves into more native k8s interface to configure the MetalLB endpoint. also allows to use different LoadBalancer than MetalLB